cli/api.py

```diff
@@ -174,9 +182,15 @@ def generate_api_v2(model_output_dir, output, aggregation_level, state, fips):
         states=active_states,
     )
     _logger.info(f"Loading all regional inputs.")
+
+    regions_data = combined_datasets.load_us_timeseries_dataset().get_regions_subset(regions)
+    test_positivity_results = test_positivity.AllMethods.run(regions_data)
+    test_positivity_results.write(test_positivity_all_methods)
```

Comment:
If we're writing the test positivity output here, it won't get included as an artifact.  Maybe this file would be better as an input and be generated during the pyseir step?

Reply:
Generating them is so fast it is hardly worth saving :-P

Reply:
It takes about 19 seconds to run all methods for all states+counties on my dinky laptop. Not great but not horrible.